oshc/main/templates/base.html

```diff
@@ -28,7 +28,7 @@
           <li class="dropdown">
             <a href="#" class="dropdown-toggle" data-toggle="dropdown">Sessions <span class="caret"></span></a>
             <ul class="dropdown-menu" role="menu">
-              <li><a href="https://www.youtube.com/channel/UC1_IAby-9me3iICtxuiDL5Q" target="_blank">View Session</a></li>
+              <li><a href="https://www.youtube.com/channel/UC1_IAby-9me3iICtxuiDL5Q/videos/" target="_blank">View Session</a></li>
```

Comment:
There is a minor glitch here, the issue needs the url to be updated to `https://www.youtube.com/channel/UC1_IAby-9me3iICtxuiDL5Q/videos` and not to `https://www.youtube.com/channel/UC1_IAby-9me3iICtxuiDL5Q/videos/`. 

Results are different 👇 

<img width="941" alt="screen shot 2017-12-28 at 10 04 44 pm" src="https://user-images.githubusercontent.com/4143778/34416831-5d136a70-ec1b-11e7-9134-2c56adc46b62.png">

<img width="921" alt="screen shot 2017-12-28 at 10 04 56 pm" src="https://user-images.githubusercontent.com/4143778/34416798-37b3f6a0-ec1b-11e7-8f70-d2576c0faab1.png">

Need to have the second one.

Reply:
Nice catch @tapasweni-pathak :+1: